Client/App.jsx

```diff
@@ -68,16 +69,33 @@ class ImageCarousel extends React.Component {
   }
 
   getImagesForEndpoint() {
+    console.log('inside endpoint');
     var productID;
     if(window.location.pathname.slice(1) === '') {
-      productID = 1;
-    } else if (window.location.pathname.slice(1) >= 101) {
-      productID = 1;
+      productID = 0;
+    } else if (window.location.pathname.slice() >= 100) {
+      productID = 0;
     } else {
       productID = window.location.pathname.slice(1);
     }

```

Comment:
I think line 76 is missing a 1 in the parentheses. Without it, you'll be including the question mark, so the expression will be something like:

else if ('?102' >= 100)

Reply:
great catch! thank you!! I have edited this now